csrc/python_frontend/fusion_cache.cpp

```diff
@@ -526,8 +529,15 @@ void FusionCache::deserialize(std::string filename) {
 
     auto fb_fec_node = fusion_cache_buffer->auto_gen_schedules()->Get(idx);
     auto fusion_schedule = queryFusionSchedules(trie_node->fusion_id);
-    fusion_schedule->auto_gen_schedules->deserialize(fb_fec_node);
+
+    // Parallelize the deserialization of each FusionExecutorCache.
+    getThreadPool()->run([=]() {
```

Comment:
nitpick: Can we add a knob to disable this. Similar to what we did with https://github.com/NVIDIA/Fuser/blob/de03e5de0713d3b1e4343c20361c7d45fba4a77c/csrc/options.cpp#L178